core/pom.xml

```diff
@@ -218,6 +218,16 @@
                 </executions>
             </plugin>
             -->
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
Spoke too soon. From what I can tell, the set of activated profiles is locked in at the parent pom--it doesn't look like you can change it in individual artifacts. Reinstating copypasta. _sad panda_